components/servo/delegate.rs

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/// Sender for errors raised by delegate request objects.
+///
+/// This allows errors to be raised asynchronously.
+pub(crate) trait DelegateErrorSender {
+    fn raise_response_send_error(&self, error: bincode::Error);
+}
```

Comment:
I have been thinking that we should move all the "responder" pattern structs to a file called `responders.rs`. Maybe this would be a good fit for that file rather than making a new one.

Reply:
done in c97fda96e5e47424b984408cfafda63df5a0c21b